common/on_actions/babylon_on_monthtly_pulse.txt

```diff
@@ -118,6 +128,13 @@ on_babylon_monthly_pulse_store_infamy = {
 			name = infamy_on_month_start
 			value = infamy
 		}
+		if = {
+			limit = {
+				# This should only pop for a singular country each month. Most likely a united China
+				has_global_highest_gdp = yes
+			}
+			babylon_vienna_congress_compile_lists = yes
```

Comment:
just to random_country, it's more efficient and more sensible.

Reply:
Yea but this is already in a random_country. If I do random_country again this would just pop 300+ times on random targets =/